src/components/Dashboard/dashboard.module.scss

```diff
@@ -0,0 +1,77 @@
+.dashboard { 
+    border-left: #dad8d8 solid 2px;
+    border-radius: 3px;
+    padding-left: 3%;
+    margin: 1%;
+}
+.dashboardContainer {
+    overflow-y: scroll;
+    overflow-x: hidden;
+    height: 800px;
+}
+
+.dashboardNav { 
+    display: flex;
+    justify-content: space-between;
+    --width: 100%;
+    padding: 1%;
+}
+
+.dashboardReportCard { 
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 1rem;
+    border: #dad8d8 solid 1px;
+    border-radius: 3px;
+    padding: 3%;
+}
+
+.dashboardTitle { 
+    width: 50%;
+    padding-top: 20px;
+    display: grid;
+}
+.dashboardScore { 
+    display: inline-flex;
+    background-color: var(--seed-color-primary);
+    border-radius: 50%;
+    padding: 10px;  
+    text-align: center;
+    margin: 8px;
+    color: #ffeaea;
```

Comment:
What color is this? if none of the existing colors work, add a new CSS variable to app.scss

hardcoding colors into stylesheets will be a big hassle to maintain in the future so we prefer CSS variables all living in a single stylesheet

Reply:
Also use rem instead of px